cri/v1alpha1/cri.go

```diff
@@ -121,7 +120,19 @@ type CriManager struct {
 
 // NewCriManager creates a brand new cri manager.
 func NewCriManager(config *config.Config, ctrMgr mgr.ContainerMgr, imgMgr mgr.ImageMgr) (CriMgr, error) {
-	streamServer, err := newStreamServer(ctrMgr, streamServerAddress, streamServerPort)
+	var streamServerAddress string
+	streamServerPort := config.CriConfig.StreamServerPort
+	// If stream server reuse the pouchd's port, extract the port from pouchd's listening addresses.
+	if config.CriConfig.StreamServerReusePort {
+		streamServerAddress, streamServerPort = extractIPAndPortFromAddresses(config.Listen)
```

Comment:
since the user maybe pass multiple tcp addresses, could we show the address and port which CRI stream uses?

Reply:
If the streamServerAddress we get is `0.0.0.0`, we will reacquire the concrete IP that the stream server will use in `newStreamServer`, so I output the log there.

Reply:
oops, my mistake.